apps/pipelines/models.py

```diff
@@ -325,6 +335,24 @@ def get_static_trigger_experiment_ids(self) -> models.QuerySet:
             .values("trigger_experiment_id")
         )
 
+    @property
+    def version(self) -> Version:
+        from apps.experiments.models import VersionFieldDisplayFormatters
+
+        reserved_types = ["StartNode", "EndNode"]
+
+        return Version(
+            instance=self,
+            fields=[
+                VersionField(name="name", raw_value=self.name),
+                VersionField(
+                    name="nodes",
+                    queryset=self.node_set.exclude(type__in=reserved_types),
+                    to_display=VersionFieldDisplayFormatters.format_nodes,
```

Comment:
Do you still need this formatter now that it's a queryset?

Reply:
Correct, thanks